### ISSUE TITLE
Make it clear project is Python3 only

### ISSUE THREAD

**Issue by geographika** [brennerm/python-apt-repo#2]
Thanks for a very useful library!

It might be worth adding a note to the readme that it is Python3 only - or adding something to setup.py e.g.  `python_requires='>=3.5'`

On Python2 there are the following errors:

```
ImportError: No module named lzma
import urllib.error
ImportError: No module named error
```

**Comment by brennerm** [comment]
Thanks for the hint. I'll think about whether to add a note or making the library compatible with Python 2.
@geographika Do you think supporting Python 2 is still necessary?

**Comment by geographika** [comment]
I'd imagine the library is used for one-off devops scripts rather than as part of a larger application, which would mean py2 compatibility is less important (for info I'm using it to check Python versions available in Debian that are installed on https://live.osgeo.org). 

I'd imagine py2 support would require a few changes (no lzma library, url functions not available etc.). A project description at https://pypi.org/project/apt-repo/ would maybe me of more benefit if you have time to spend on the project. 

Thanks again for your work.

**Comment by brennerm** [comment]
Fixed with ee3f24674aeb7615a61d728ca352e541f5d8f169.
